cmd/libs/go2idl/client-gen/testoutput/clientset_generated/test_release_1_1/clientset.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test_release_1_1
```

Comment:
Ah, sorry I forgot to mention there is a hack/update-codegen.sh that update all the generated code. https://github.com/kubernetes/kubernetes/blob/master/hack/after-build/run-codegen.sh#L36

As for the end-user, when we release the tool, we'll give them examples so they won't be baffled.